src/database/simple.jl

```diff
@@ -394,6 +394,34 @@ const renames = [
     @add_rename iteratoreltype IteratorEltype   v"0.7.0-DEV.3309"
     @add_rename nb_available   bytesavailable   v"0.7.0-DEV.3477"
     @add_rename broadcast_indices broadcast_axes  v"0.7.0-DEV.4936"
+    @add_rename Base.Test      Test             v"0.7.0-alpha" 
```

Comment:
This is already taken care of

Reply:
OK, I will see why it didn't trigger on the depoexercism.jl FC PRs.